Applications/Calculator/CalculatorWidget.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <AK/Vector.h>
+#include <LibGUI/GWidget.h>
+#include "Calculator.h"
+#include "Keypad.h"
+
+class GTextBox;
+class GButton;
+class GLabel;
+
+class CalculatorWidget final : public GWidget {
+public:
+    explicit CalculatorWidget(GWidget*);
+    virtual ~CalculatorWidget();
+
+    virtual const char* class_name() const override { return "CalculatorWidget"; }
```

Comment:
Instead of manually implementing `class_name()`, please use the `C_OBJECT` macro.
The point of `C_OBJECT` is to make it easy to add more auto-generated members of `CObject`s in the future :)

Reply:
Right, somehow forgot about that. Will fix :smiley: